script.js

```diff
@@ -1,54 +1,74 @@
-const bookTitle = document.getElementById('book-title');
-const bookAuthor = document.getElementById('book-author');
-const addBtn = document.getElementById('btn');
-const listEntry = document.getElementById('list-entry');
-
-let books = JSON.parse(localStorage.getItem('books')) || [];
-
-const addBook = () => {
-  const title = bookTitle.value.trim();
-  const author = bookAuthor.value.trim();
-  if (title.length > 0 && author.length > 0) {
-    const book = { title, author };
-    books.push(book);
-    localStorage.setItem('books', JSON.stringify(books));
-    title.value = '';
-    title.author = '';
+class Book {
+  constructor() {
+    this.bookTitle = document.getElementById('book-title');
+    this.bookAuthor = document.getElementById('book-author');
+    this.addBtn = document.getElementById('btn');
+    this.listEntry = document.getElementById('list-entry');
+
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+
+    this.addBtn.addEventListener('click', this.addBook.bind(this));
+
+    this.renderBooks();
   }
-  this.location.reload();
-};
-
-const updateStorage = () => {
-  localStorage.setItem('books', JSON.stringify(books));
-};
-
-const removeBook = (title, author) => {
-  books = books.filter((book) => book.title !== title && book.author !== author);
-  updateStorage();
-};
-
-addBtn.addEventListener('click', addBook);
-
-listEntry.innerHTML = `${books
-  .map(
-    (book) => `
-      <li class="book-card">
-        <p class="book-title">${book.title}</p>
-        <p class="book-title">${book.author}</p>
-        <button class="remove-btn">Remove</button>
-      </li>
-      <hr />`,
-  )
-  .join('')}`;
-
-const removeBtns = document.querySelectorAll('.remove-btn');
-
-removeBtns.forEach((btn) => {
-  btn.addEventListener('click', (e) => {
-    const card = e.target.closest('.book-card');
-    const title = card.querySelector('.book-title');
-    const author = card.querySelector('.book-author');
-    removeBook(title.innerText, author);
-    this.location.reload();
-  });
-});
+
+  addBook() {
+    const title = this.bookTitle.value.trim();
+    const author = this.bookAuthor.value.trim();
+
+    if (title.length > 0 && author.length > 0) {
+      const book = { title, author };
+      this.books.push(book);
+      this.updateStorage();
+      this.clearInputs();
+      this.renderBooks();
+    }
+  }
+
+  updateStorage() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  removeBook(title, author) {
+    this.books = this.books.filter(
+      (book) => book.title !== title && book.author !== author,
+    );
```

Comment:
- [ ] The removeBook method removes all books thathave the same name & author. Please make sure the only removed book is the one selected. 
- [ ] I suggest you implement this by assigning each a book a unique id and then target the book by its id. 
- [ ] You can also validate and make sure no book with the same name & author can be entered twice.